#!/usr/bin/env python
PACKAGE = "franka_panda_controller_swc"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("translational_stiffness", double_t, 0, "Cartesian translational stiffness", 500, 0, 500)
gen.add("rotational_stiffness", double_t, 0, "Cartesian rotational stiffness", 50, 0, 50)
gen.add("nullspace_stiffness", double_t, 0, "Stiffness of the joint space nullspace controller (the desired configuration is the one at startup)", 80, 0, 100)

exit(gen.generate(PACKAGE, "dynamic_compliance", "compliance_param"))
